tests/setup.sh

```diff
@@ -22,32 +22,11 @@ rpmdb --initdb
 rpmhome=/root/.config/rpm
 mkdir -p ${rpmhome}
 
-# setup default signing id + key
-sqemail="rpmbuild-user@$(uname -n)"
-sqhome=${rpmhome}/sq
-sqkey=$(sq key generate \
-   --batch \
-   --quiet \
-   --own-key \
-   --without-password \
-   --can-sign \
-   --cannot-authenticate \
-   --cannot-encrypt \
-   --email ${sqemail} \
-      2>&1 | awk '/Fingerprint/{print $2}')
-
-cat << EOF > ${rpmhome}/macros
-%_openpgp_autosign_id ${sqkey}
-%_openpgp_sign sq
-EOF
-
-# import the signing key by default
-sq cert export \
-    --cert-email "${sqemail}" > /root/rpm-key.asc
-rpmkeys --import /root/rpm-key.asc
-
 # gpg-connect-agent is very, very unhappy if this doesn't exist
 mkdir -p /root/.gnupg
 chmod 700 /root/.gnupg
 
-
+# setup default signing id + key (USER is not set when this runs!)
```

Comment:
I wonder if we should actually handle the lack of USER value in the script as well. It seems like podman/OCI containers at least don't have it set by default:
```
podman run -it --rm fedora sh -c 'echo $USER'
```

Reply:
... which is why we set USER explicitly below, of course (since this very script runs in a podman container) :smile:

Reply:
Added a bail-out on missing $USER too.